package.json

```diff
@@ -4,7 +4,8 @@
   "description": "An iteration of the Node.js core streams with a series of improvements",
   "main": "index.js",
   "dependencies": {
-    "fast-fifo": "^1.0.0"
+    "fast-fifo": "^1.0.0",
+    "queue-microtask": "^1.2.3"
```

Comment:
Seems queueMicrotask is supported since Node 11, so I'm fine just relying on it being there, unless there is some other reason to import it as well?

Reply:
nope, just for backwards compatibility, should i remove it?

Reply:
Yea, let's do it